### DISCUSSION TITLE
reolink smart AI automation switches

### DISCUSSION THREAD

**Discussion by gtalusan** [koush/scrypted#1334] [Ideas]
Hello, I have a Reolink POE Doorbell and I'm using the reolink plugin.  I can see that it generates events for FaceDetect, PeopleDetect, VehicleDetect, and CatDogDetect.  That information seems to be lost (at least to me) when Scrypted generates events.  I took a peek at the code and the AI events seem to be passed over to the objectdetector.  After that, I'm not sure what happens.

I can see the onvif plugin can trigger a doorbell event given an ONVIF event name.

It'd be a nice feature if the {Face/People/Vehicle/CatDog}Detect events were able to trigger binary state switches so I can set off my own automations.  Is this possible with other plugins in Scrypted?

**Comment by koush** [comment]
yes use the video analysis plugins smart motion sensor

also , the doorbell only detects people.

**Comment by gtalusan** [comment]
Ahhh, I see.. okay, it was a bit confusing to set up but I understand now.  Awesome, thanks so much for your quick response and help.
